checks.py

```diff
@@ -19,7 +19,12 @@ def function_separation(filename: str, content: str) -> list:
 
 def leading_line(filename: str, content: str) -> list:
     name = "G9 - Leading/Trailing lines"
-
-    if content.startswith('\n') or content.endswith('\n\n'):
-        return [{"type": "minor", "file": filename, "line": 1, "name": name}]
-    return []
+    line = []
+    if content.startswith('\n'):
+        line.append(1)
+    if content.endswith('\n\n'):
+        line.extend([content.count('\n'), content.count('\n')+1])
+    if len(line) > 0:
+        line = (', '.join(map(str, line)))
```

Comment:
If both leading and trailing lines are present, please return an array with 2 elements (with the same content except the line) instead of multiples lines in the same element.

Reply:
I've made a few changes that should result in the output at line: display an array of the leading line number and trailing line number.
Suppose we have a file with leading and trailing spaces (trailing at line 11 as an example), the output would be returned as "line": [ 1 , 11 ] . Is this what you were expecting?

Reply:
Not exactly, with the current logic of the other parts of the code I would need the function to return an array with two distinct elements, so in your example case it should be `[{"type": "minor", "file": filename, "line": 1, "name": name}, {"type": "minor", "file": filename, "line": 11, "name": name}]`.

Reply:
Ohh I see, I now understand the return result. I'll work on it right away :)

Reply:
I've changed the function to return an array with two distinct elements as you described.

Reply:
Looks great ! Thanks again for your contribution :)